From 34098908ca98dde826da021ec0806c2203fe8310 Mon Sep 17 00:00:00 2001
From: Jordan Hendricks <jordan.hendricks@joyent.com>
Date: Fri, 23 Jun 2017 18:58:46 +0000
Subject: [PATCH] MANTA-3296 Mako returned different md5 sums for the same MPU
 object Reviewed By: Robert Mustacchi <rm@joyent.com> Approved By: Robert
 Mustacchi <rm@joyent.com>

---
 deps/nginx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/nginx b/deps/nginx
index 747b5c0..de2e377 160000
--- a/deps/nginx
+++ b/deps/nginx
@@ -1 +1 @@
-Subproject commit 747b5c0807f3cd5cd3967adccb09d722a039e515
+Subproject commit de2e377384c2b4d61eed75bbd289d07558ab0bbc
-- 
2.21.0

